src/transformers/models/bit/image_processing_bit.py

```diff
@@ -257,23 +258,24 @@ def preprocess(
 
         images = make_list_of_images(images)
 
-        if not valid_images(images):
+        if images and not valid_images(images):
```

Comment:
What's the reason for adding the `if images` logic here? I don't think `images` should ever be false here.

Reply:
Mostly because in some places image could be `None`! But `valid_images` should take care of that either way, will fix